go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/qdm12/ddns-updater
 go 1.21
 
 require (
-	github.com/breml/rootcerts v0.2.16
+	github.com/breml/rootcerts v0.2.15
```

Comment:
It's still there 😢
You can just go get the version 0.2.16 and commit it I think (anyway it'll get squashed)

Reply:
What 😡 I'm clearly doing something wrong I guess. Let me look at it again.

Reply:
Well if that didn't work, then I don't know what to do. Everything I'm seeing for the `global_val` branch is 0.2.15 🤷